lldb/tools/lldb-dap/LLDBUtils.cpp

```diff
@@ -13,16 +13,22 @@ namespace lldb_dap {
 
 void RunLLDBCommands(llvm::StringRef prefix,
                      const llvm::ArrayRef<std::string> &commands,
-                     llvm::raw_ostream &strm) {
+                     llvm::raw_ostream &strm, bool parse_command_directives) {
```

Comment:
add a `bool &fatal_error` that is an out parameter to indicate we got an error on a `!` command, clients will then use this to figure out what to do instead of aborting. See inline comments below